prow/cluster/jobs/istio/pkg/OWNERS

```diff
@@ -0,0 +1,3 @@
+approvers:
+  - geeknoid
+  - ozben
```

Comment:
I don't think this is the right name?

Reply:
Whoops, I used his Google alias rather than his GitHub alias.